src/Service/HttpViewManagerConfigTrait.php

```diff
@@ -22,11 +22,15 @@ private function getConfig(ContainerInterface $container)
     {
         $config = $container->has('config') ? $container->get('config') : [];
 
-        if (isset($config['view_manager'])
-            && (is_array($config['view_manager'])
-                || $config['view_manager'] instanceof ArrayAccess
-            )
-        ) {
+        if (! isset($config['view_manager'])) {
+            return [];
+        }
+
+        if ($config['view_manager'] instanceof ArrayAccess) {
```

Comment:
Thre is a problem with the `config` service in `zend-mvc`. Somewhere is declared as `array|ArrayAccess`, but then is passed on methods that accept just an `array`. This will be a problem with strict types.

I would prefer to return just an `array`. You're right, it's dangerous. What about to convert it iterating the array?

Reply:
I know. Do you agree that should be better to return just an array? In order to use strict types in the future I think should be better IMHO.

I think it's not possible to convert safely an object implementing `ArrayAccess` in `array`, so I should revert it, but it's a problem to handle it in other classes

Reply:
The return value was used only by private methods, so I changed the signature to allow `array|ArrayAccess`